metadata/cube.xml

```diff
@@ -11,17 +11,17 @@
 			<option name="activate" type="button">
 				<_short>Activate</_short>
 				<_long>Activates the cube with the specified button.</_long>
-				<default>&lt;alt&gt; &lt;ctrl&gt; BTN_LEFT</default>
+				<default>&lt;super&gt; &lt;shift&gt; BTN_LEFT</default>
```

Comment:
NAK. Ctrl+Alt+Drag should be the default.

Reply:
It matches well with the config as a whole.  <kbd>Super</kbd> being the default modifier.

https://github.com/WayfireWM/wayfire/wiki/Tutorial#default-key-bindings